src/commands/commands/playbook/commands/template/commands/list/playbookTemplateList.ts

```diff
@@ -1,5 +1,57 @@
-import { Command } from "commander";
+import path from "path";
+import { ConfigValue } from "../../../../../../../constants";
+import { Command } from "../../../../../../../extensions";
+import { RepoServiceFactory } from "../../../../../../../factories";
+import { CseCliConfig, templateItem } from "../../../../../../../models";
+import { RepoServiceProvider } from "../../../../../../../services";
+import { Config } from "../../../../../../../utils";
+
+export interface PlaybookTemplateListOptions {
+  branch: string;
+  filename: string;
+  filepath: string;
+  githubToken: string;
+}
 
 export const playbookTemplateList = new Command()
   .name("list")
-  .description("Not yet implemented");
+  .description("List available templates.")
+  .option("-b, --branch <branch>", "Branch of playbook to use")
+  .option("-f, --filename <filename>", "Name of template file", "templates.json")
+  .option("-g, --github-token <github-token>", "GitHub personal access token")
+  .option("-p, --filepath <filepath>", "Path to template file in playbook", path.sep)
+  .addAction(async (options: PlaybookTemplateListOptions, config: CseCliConfig) => {
```

Comment:
In my recent PR #33, I added some basic dependency injection that handles all of this service instantiation. You would just need to have something like:

```
.addAction(async (collection: ServiceCollection...) => {
  const { playbookService } = collection;
  // Have function in `PlaybookService` that retrieves all templates from repo
```